crates/uv/tests/help.rs

```diff
@@ -634,6 +634,63 @@ fn help_with_version() {
     ----- stdout -----
     uv [VERSION] ([COMMIT] DATE)
 
+    ----- stderr -----
+    "###);
+}
+
+#[test]
+fn test_with_no_pager() {
+    let context = TestContext::new_with_versions(&[]);
+
+    uv_snapshot!(context.filters(), context.help().arg("--no-pager"), @r###"
```

Comment:
We never page for the root command so `--no-pager` should have no effect here (we also can't page during snapshots so it's kind of moot).

Reply:
Should I remove the test then?

Reply:
Seems nice to confirm that the option exists and works. Maybe just add a comment noting that we can't actually check if the pager is used?